roles/kubernetes/node/defaults/main.yml

```
@@ -92,6 +92,7 @@ loadbalancer_apiserver_memory_requests: 32M
 loadbalancer_apiserver_cpu_requests: 25m
 
 loadbalancer_apiserver_keepalive_timeout: 5m
+loadbalancer_apiserver_pod_name: {% if loadbalancer_apiserver_type == "nginx" %}nginx-proxy{% else %}haproxy{% endif %}
```

Ah wait could you add double quotes as well like so
```suggestion
loadbalancer_apiserver_pod_name: "{% if loadbalancer_apiserver_type == "nginx" %}nginx-proxy{% else %}haproxy{% endif %}"
```

Otherwise looks good to me thanks!